.gitignore

```diff
@@ -7,4 +7,5 @@ build/
 __pycache__
 dist/
 venv
+tags
```

Comment:
Lots of devs (me included) use ctags when developing. This isn't strictly necessary, but helps me generate and use ctags without worrying about checking them in. I could remove it if you'd prefer.